.changeset/grumpy-toys-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/cli": patch
+---
+
+- Removed descriptions adding for x-tagGroups for the `join` command. Descriptions in x-tagGroups are not supported and cause errors on linting.
+- `info.title` is now used as a name in x-tagGroups instead of a file name for the `join` command to make possible the joining of files with the same names.
```

Comment:
```suggestion
- Updated `info.title` to be used as a name in x-tagGroups instead of a file name for the `join` command, so you can now join files with the same names.
```

Reply:
Fixed, thanks!